### ISSUE TITLE
Title Permutations including Episode Names

### ISSUE THREAD

**Issue by mcteo** [SiCKRAGETV/sickrage-issues#785]
Hey. First off, SickRage is awesome. Great Work!

I was looking through the logs trying to debug a separate issue with providers (and some providers blocked by my ISP), and saw that SickRage tends to try different permutations of the file name, often including the episode name itself. Since I have rarely seen any uploaders that use the actual individual episode name, I think these requests might be wasted. For example, much more useful would be to try search for both "S??E??" and "?x??". I could change this myself, but am just wondering is there any reason for not doing so?

**Comment by fernandog** [comment]
@mcteo it already does that
